config/api_resources/Order.xml

```diff
@@ -0,0 +1,412 @@
+<?xml version="1.0" ?>
```

Comment:
Sur tous les fichiers provenant de Sylius à la base, on a une bonne pratique :
- Ajouter un commentaire tout en haut du fichier indiquant d'ou vient le file originalement. Ex `<!-- This file override the origininal Sylius file /path/to/sylius/file`
- Ajouter un commentaire au niveau des lignes qu'on a changé
Ex ```
<!-- START of the line updated from the original file

END of the lines updated from the original file-->
```